optional_requirements_extra_data_sources.txt

```diff
@@ -1,3 +1,3 @@
 boto3 >= 1.9.0
```

Comment:
The idea behind these dependencies is that they should be install-abled by default, so feel free to add them to `requirements.txt` instead of here (and also make the datasources they pretain to imported in the main __init__.py by default, rather than inside a try/except)